addons/marker_flags/stringtable.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Marker_Flags">
+        <Key ID="STR_ACE_Marker_Flags_DisplayName_Settings">
+            <English>Marker Flag</English>
+            <German>Markierungsfahnen</German>
+        </Key>
+        <Key ID="STR_ACE_Marker_Flags_placeAnywhere_DisplayName">
```

Comment:
I think use PascalCase for stringtable entries, here and below (in code as well).